aws/client.go

```diff
@@ -0,0 +1,63 @@
+package aws
+
+import (
+	"github.com/ONSdigital/go-ns/log"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/s3"
+	"io/ioutil"
+	"github.com/aws/aws-sdk-go/aws"
+	"bytes"
+	"io"
+)
+
+const awsBucketName = "dp-csv-splitter-1"
```

Comment:
These can be left as they are for now but I would speak with @LloydGriffiths to see if we want to make any variable (or here `const`) pertaining to AWS be configurable from the environment.